apps/src/templates/publishDialog/PublishDialog.jsx

```diff
@@ -42,29 +50,36 @@ class PublishDialog extends Component {
 
   render() {
     return (
-      <Dialog
-        title={i18n.publishToPublicGallery()}
+      <BaseDialog
         isOpen={this.props.isOpen}
         handleClose={this.close}
+        useUpdatedStyles
```

Comment:
What does this do?

Reply:
https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true

Reply:
Sorry — what is the purpose of passing this prop?

Reply:
Without `useUpdatedStyles`
<img width="650" alt="screen shot 2018-06-26 at 12 57 33 pm" src="https://user-images.githubusercontent.com/12300669/41936015-9a0d9344-7940-11e8-8c40-f449be15dd5f.png">
The style differences are that without `useUpdatedStyles` we end up with the old style purple x in the corner to dismiss the dialog, the bottom corners are rounded and the width shrinks some.

Oddly, I also think using the new styles removes the `.modal-body` class that the UI tests are expecting to find.

Ideally, we probably want: 
- gray x to dismiss 
- wider width 
- 4 consistently rounded corners 
- modal-body class for all `BaseDialogs` 
- no reliance on global styles 
- remove both `useUpdatedStyles` and `useDeprecatedGlobalStyles` props in favor of uniform, consistent styling 

`BaseDialog` needs to be updated to new ES6 syntax anyway, so I'm going to check with Mark about what design we actually want and then update that component as follow up.

Reply:
> remove both useUpdatedStyles and useDeprecatedGlobalStyles props in favor of uniform, consistent styling

🎉

Not blocking this PR of course, but that sounds like awesome follow up!